src/server/qgsstorebadlayerinfo.h

```diff
@@ -19,19 +19,20 @@
 #define QGSSTOREBADLAYERINFO_H
 
 #include "qgsprojectbadlayerhandler.h"
+#include "qgis_server.h"
 #include <QStringList>
 
 /**
  * \ingroup server
  * Stores layer ids of bad layers
  * \since QGIS 3.6
  */
-class QgsStoreBadLayerInfo: public QgsProjectBadLayerHandler
+class SERVER_EXPORT QgsStoreBadLayerInfo: public QgsProjectBadLayerHandler
 {
   public:
     /*Default constructor
      */
-    QgsStoreBadLayerInfo() = default;
+    QgsStoreBadLayerInfo(){}
```

Comment:
I think it's the missing extra-asterisk for doxygen formatting that makes travis complain

/**
 * Construct a new QgsStoreBadLayerInfo
 */